src/Controller/DigitalBrochureController.php

```diff
@@ -41,8 +41,9 @@ public function request($id, $slug, Request $request)
 
         switch ($sanitisedSlug) {
             case "commercial-agreements-spring-2021-digital-brochure":
```

Comment:
This should have a return statement too

Reply:
Thanks, the return statement is for both cases. I thought i'd put it into one statement rather than repeat it again. I think that both of these cases should be going to the new digital brochure? what do you think?

Reply:
Haha no worries. Me neither had to google it. Thanks.